### DISCUSSION TITLE
[FEATURE REQUEST] Application Folders

### DISCUSSION THREAD

**Discussion by SkylerMalinowski** [linuxdeepin/developer-center#2853] [Features Request & Ideas | 特性请求 & 头脑风暴]
[Purpose]
Graphical management of installed applications.

[Implementation]
Drag-and-drop one application icon on to another application icon, as seen in Android, iOS, Windows, and MacOS.

**Comment by uiYzzi** [comment]
Yes, it’s absolutely necessary!!!

**Comment by uiYzzi** [comment]
@wangjia12
